tests/testthat/test-topology.R

```diff
@@ -41,3 +41,37 @@ test_that("automorphism_group works with colored graphs", {
   aut <- aut[order(sapply(aut, "[[", 1))]
   expect_that(aut, equals(list(c(1, 4, 3, 2), c(3, 2, 1, 4))))
 })
+
+test_that("isomorphisms works", {
+  motif <- make_empty_graph(directed = FALSE) +
+    vertices("D1", "D2", type = c("type1", "type1")) +
+    edges("D1", "D2", type = c("type2"))
+  subgraph_isomorphisms(
+    target = motif, pattern = motif, method = "vf2",
+    vertex.color1 = 2:1, vertex.color2 = 1:2
+  )
+})
```

Comment:
The `type` attribute is unused here. It's not useless to test with `target` and `pattern` the same, but normally they should be different, and `pattern` should be smaller.

For example, let `target` be a `make_ring(10,circular=T)` and `pattern` a `make_ring(5, circular=F)`.